mpdel-tablist.el

```diff
@@ -108,7 +108,8 @@
 
 (navigel-method mpdel navigel-entity-to-columns ((song libmpdel-song))
   (vector
-   (propertize (or (libmpdel-entity-name song) "") 'face 'mpdel-tablist-song-name-face)
+   (propertize (or (libmpdel-entity-name song)
+                   (libmpdel-song-file song) "") 'face 'mpdel-tablist-song-name-face)
```

Comment:
I suggest improving `libmpdel-entity-name` for songs in libmpdel instead.

Reply:
I want to make sure I understand correctly, the change for defaulting the song name should go in `libmpdel-entity-name` in the [libmpdel](https://github.com/mpdel/libmpdel) project?

Reply:
I'm submitting a PR for libmpdel.  Feel free to close this, if that's the desired outcome.

Reply:
:+1: